core/src/components/toolbar/toolbar.tsx

```diff
@@ -88,19 +147,20 @@ export class Toolbar implements ComponentInterface {
   }
 
   render() {
-    const theme = getIonTheme(this);
+    const theme = getIonTheme(this.el);
```

Comment:
```suggestion
    const theme = getIonTheme(this);
```

Is this change necessary? I see the same theme logged checking `this`.